dolphinscheduler-master/src/main/docker/Dockerfile

```diff
@@ -25,6 +25,6 @@ WORKDIR $DOLPHINSCHEDULER_HOME
 
 ADD ./target/master-server $DOLPHINSCHEDULER_HOME
 
-EXPOSE 12345
+EXPOSE 5678 5679
```

Comment:
Should remain unchanged?

Reply:
The worker should expose 5678/5679, 12345 is used by api.